gravity.sh

```diff
@@ -121,19 +115,19 @@ gravity_store_in_database() {
   local timestamp
   timestamp="$(date --utc +'%s')"
   local inputfile
-  if [ "$table" == "whitelist" ] || [ "$table" == "blacklist" ] || [ "$table" == "regex" ]; then
-    # Apply format for white-, blacklist, and regex tables
+  if [[ "${table}" == "gravity" ]]; then
+    # No need to modify the input data for the gravity table
+    inputfile="${source}"
+  else
+    # Apply format for white-, blacklist, regex, and adlists tables
     # Read file line by line
     grep -v '^ *#' < "${source}" | while IFS= read -r domain
```

Comment:
Does this properly ignore empty lines?

Reply:
I added some code which will ensure that empty lines won't get added to the import file we pass to the `sqlite3` command.